src/atoms/Button/Button.tsx

```diff
@@ -31,9 +31,16 @@ const StyledButton = styled(BasicButton)<{
   color: ${props => (props.secondary ? props.theme.primary : 'white')};
   font-size: ${props => props.large && scale(1)};
   ${padding(scale(-1), scale(2))};
-  ${transitions(['opacity', 'background', 'color'], transitionDuration)};
+  ${transitions(
+    ['opacity', 'background', 'color', 'box-shadow'],
+    transitionDuration,
+  )};
   user-select: none;
 
+  :focus {
+    box-shadow: 0 0 0 2px white, 0 0 0 4px ${props => props.theme.primary};
```

Comment:
Let's use em instead of pixels, for example 2px = 2/16em = 0.125em

Reply:
Good catch!